portfolio/src/main/webapp/script.js

```diff
@@ -15,6 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
+
+
+var projectCounter = 0;
+var projectList = [
+    ["Gitlet", "This project was my favorite and final project for my CS61B class at Berkeley. The gist of the project was to recreate the popular version-control system: Git, but a bit more simplified."],
+    ["Link!", "My first team-developed project, developed during Google's 2019 CSSI program. We developed a web application that determined mutual availaibilites among friends and found events for everyone to hang out using differnt API's."],
+    ["Simon", "My final project in my first coding class. Developed a web application that mimicked the color-sequence memorization game: Simon."]
+    ];
+
+
```

Comment:
it looks like you put this below the docs for `addRandomGreeting`. please move above.

Reply:
By move above do you mean put it below? Cuz for me those lines are above addRandomGreeting